src/components/GaugeChart/handleSeries.js

```
@@ -435,6 +435,74 @@ function handleOther(iChartOption, seriesUnit, series, data) {
   }
 }
 
+function handleStatus(seriesUnit, iChartOption){
+  let status = iChartOption.data[0].status;
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)